instagram/src/components/CommentSection/CommentsContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './CommentSection.css';
+
+import Comment from './Comment';
+
+const CommentsContainer = props => {
+    return (
+        <>
+            {props.comments.map((c, index) => <Comment key={index} comment={c}/>)}
+        </>
```

Comment:
solid use of the empty wrapper that reacts like React.Fragment to keep the DOM tree clean